packages/formulas/formulas.js

```diff
@@ -0,0 +1,103 @@
+
+//Phy
+//Speed/Velocity
+export function spd(){    
```

Comment:
Remove `export` from all the functions & at the end of the file add the following code:
```js
module.exports = {
  spd,
  acc,
  momen,
  // Add all the function names here
}
```